### ISSUE TITLE
Translations are not loaded sometimes

### ISSUE THREAD

**Issue by ovalseven8** [threema-ch/threema-web#542]
## Expected Behavior
Translation is displayed

## Current Behavior
Translation is not displayed sometimes. This happened two times already, today. Sometimes it works. No idea why that is.

```
troubleshooting.PLEASE_UPDATE_APP
troubleshooting.USE_ARCHIVE_VERSION
```

![bildschirmfoto_2018-08-02_23-07-25](https://user-images.githubusercontent.com/8258609/43611386-3ebbfe28-96a9-11e8-9cc5-95264a297a8b.png)

## Your Environment
* Threema Web version: 2.0.0
* Threema App version: 3.5

**Comment by dbrgn** [comment]
Hm, that happens if the template is loaded before the translation system is ready.

**Comment by dbrgn** [comment]
This should have been fixes in #676.
